src/tools/kdb/ls.cpp

```diff
@@ -21,12 +21,9 @@ LsCommand::LsCommand () : kdb (root)
 {
 }
 
-int LsCommand::execute (Cmdline const & cl)
+int LsCommand::execute (const Cmdline & cl)
```

Comment:
> I used const before the class type all the time, also in kdb complete. Should i change it too in there?

It is not urgent, please not within this PR. Maybe together with #1328 or even seperately.

>and what about something like const Key key (without &), should it also be Key const key?

It actually does not make much sense to do that, at least not like you did it. It will increase/decrease the ref counter unnecessarily.

Also `const int` as formal parameter is of little use, the value is copied anyway, why protect it from writing to it?

The position of const seems to be inconsistent at some other places, e.g. `const char *`, maybe one of the llvm tools can change it automatically?

Btw. it is good that you use `const` so frequently.